src/OrdersList/OrdersList.js

```diff
@@ -278,7 +278,7 @@ class OrdersList extends Component {
           searchableIndexes={translatedSearchableIndexes}
           onChangeIndex={this.changeSearchIndex}
           selectedIndex={get(resources.query, 'qindex')}
-          visibleColumns={['poNumber', 'vendorCode', 'workflowStatus', 'orderType', 'created', 'owner', 'assignedTo']}
+          visibleColumns={['poNumber', 'vendorCode', 'workflowStatus', 'orderType', 'created', 'assignedTo']}
```

Comment:
please also remove `'owner': ({ owner = '' }) => owner,` from this file (249th line)